catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/ResourceRequestByDerivedUri.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.operation.impl;
+
+import ddf.catalog.data.types.Core;
+import ddf.catalog.operation.ResourceRequest;
+import java.io.Serializable;
+import java.net.URI;
+import java.util.Map;
+
+public class ResourceRequestByDerivedUri extends OperationImpl implements ResourceRequest {
```

Comment:
Where is this class used? Could use some javadoc as well

Reply:
Will be used in Alliance by the chipping service, who needs to download the derived overview product from NITFs.